app/views/locations/show.html.erb

```diff
@@ -0,0 +1,28 @@
+<% title = @location.name %>
+<% content_for :title, title %>
+
+<% content_for :before_content do %>
+  <%= govuk_back_link(
+    text: "Back",
+    href: send("locations_path", @scheme),
+  ) %>
+<% end %>
+
+<%= render partial: "organisations/headings", locals: { main: @location.postcode, sub: @location.name } %>
+
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds-from-desktop">
+        <%= govuk_summary_list do |summary_list| %>
+            <% @location.display_attributes.each do |attr| %>
+                <%= summary_list.row do |row| %>
+                <% row.key { attr[:name] } %>
+                <% row.value { attr[:name].eql?("Status") ? status_tag(attr[:value]) : details_html(attr) } %>
+                <% row.action(text: "Change", href: location_edit_name_path(location_id: @location.id, id: @scheme.id)) if attr[:edit] %>
+                <% end %>
+            <% end %>
+        <% end %>
+    </div>
+</div>
+<% if !Rails.env.production? %>
```

Comment:
Please could this live in a method in `FeatureToggle` so that we can share it?

Reply:
Of course!